package.json

```diff
@@ -10,6 +10,9 @@
     "build:tsoa": "tsoa spec-and-routes",
     "build": "tsoa spec-and-routes && tsc",
     "start": "node build/index.js",
+    "start:persona:01": "API_SWAGGER_BG_COLOR='#e4f2f3' API_SWAGGER_TITLE=API-Heidi    API_SWAGGER_HEADING=API-Heidi     node build/index.js",
```

Comment:
Do we need 'API-xxx' here or just the names 

i.e. 'Heidi' vs. 'API-Heidi' - I don't think the 'API-' adds anything but it's not that important

Reply:
Up to you .. i can change it ..